certbot/certbot/util.py

```diff
@@ -61,6 +61,19 @@
 _LOCKS = OrderedDict() # type: OrderedDict[str, lock.LockFile]
 
 
+def env_no_snap_for_external_calls():
+    """
+    Returns a modified env to pass to Popen
```

Comment:
nit: Especially since this is part of our public API and will be documented [here](https://certbot.eff.org/docs/api/certbot.util.html), can we flesh out this docstring?

Reply:
Of course, I figured there'd be some changes and wanted to address those before writing up docs and changelog additions.